docs/common/DEBUGGING.md

```diff
@@ -154,6 +186,55 @@ return { addItem };
 
 **Why:** Streams aren't created directly - they're the result of binding a handler with state. The bound handler IS the stream that can receive events.
 
+### Hover Events Not Supported
+
+**Error:** "Property 'onMouseEnter' does not exist on type..."
+
+Hover events (`onMouseEnter`, `onMouseLeave`) are not supported on div elements:
+
+```typescript
+// ❌ Hover events don't work
+<div onMouseEnter={handler}>Hover me</div>
+
+// ✅ Use click handlers instead
+<button
+  style={{ background: "transparent", border: "none", cursor: "pointer" }}
+  onClick={handleSelect}
+>
+  Click to select
+</button>
+```
+
+**Why:** The framework JSX types don't include these events (marked as `@TODO`).
+**Benefit:** Click-based interactions are more mobile-friendly anyway.
```

Comment:
I had deliberately left hover events out due to the mobile support aspect, hover events can also create dark patterns (things jumping in front of other elements unexpectedly)